service/experiment-requests.yaml

```diff
@@ -16,6 +16,26 @@
 # You can run "make presubmit" to do basic validation on this file.
 # Please add new experiment requests towards the top of this file.
 
+- experiment: 2020-09-26
+  fuzzers:
+    - aflplusplus
```

Comment:
can you please have
- aflcc
- klee
- aflcc_no_orig_dic
in there and remove all the others. Also, please change 2020-09-26 to 2020-09-20

Reply:
Done.